docs/concepts/storage/persistent-volumes.md

```diff
@@ -203,6 +206,13 @@ Generally, a PV will have a specific storage capacity.  This is set using the PV
 
 Currently, storage size is the only resource that can be set or requested.  Future attributes may include IOPS, throughput, etc.
 
+### Volume Mode
+
```

Comment:
Can you add this alpha tag to the beginning of each new section?  Then you don't need the alpha note.

```
{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
```